guides/periodic_jobs.md

```diff
@@ -0,0 +1,119 @@
+# Periodic Jobs
+
+Oban's [cron plugin](`Oban.Plugins.Cron`) registers workers that run on a cron-like schedule and enqueues jobs automatically.
+
+Periodic jobs are declared as a list of `{cron, worker}` or `{cron, worker, options}` tuples:
+
+```elixir
+config :my_app, Oban,
+  repo: MyApp.Repo,
+  plugins: [
+    {Oban.Plugins.Cron,
+     crontab: [
+       {"* * * * *", MyApp.MinuteWorker},
+       {"0 * * * *", MyApp.HourlyWorker, args: %{custom: "arg"}},
+       {"0 0 * * *", MyApp.DailyWorker, max_attempts: 1},
+       {"0 12 * * MON", MyApp.MondayWorker, queue: :scheduled, tags: ["mondays"]},
+       {"@daily", MyApp.AnotherDailyWorker}
+     ]}
+  ]
+```
+
+The crontab would insert jobs as follows:
+
+  * `MyApp.MinuteWorker` — Inserted once every minute
+  * `MyApp.HourlyWorker` — Inserted at the first minute of every hour with custom arguments
+  * `MyApp.DailyWorker` — Inserted at midnight every day with no retries
+  * `MyApp.MondayWorker` — Inserted at noon every Monday in the "scheduled" queue
+  * `MyApp.AnotherDailyWorker` — Inserted at midnight every day with no retries
+
+The crontab format respects all [standard rules][cron] and has one minute resolution. Jobs are
+considered unique for most of each minute, which prevents duplicate jobs with multiple nodes and
+across node restarts.
+
+Like other jobs, recurring jobs will use the `:queue` specified by the worker module (or
+`:default` if one is not specified).
+
+## Cron Expressions
+
+> #### Crontab Guru {: .tip}
+>
+> Cron syntax can be hard to write and read. We recommend using a tool
+> like [Crontab Guru][guru] to make sense of cron expressions and write
+> new ones.
+
+Standard cron expressions are composed of rules specifying the minutes, hours, days, months and
+weekdays. Rules for each field are comprised of literal values, wildcards, step values or ranges:
+
+  * `*` — Wildcard, matches any value (0, 1, 2, …)
+  * `0` — Literal, matches only itself (only 0)
+  * `*/15` — Step, matches any value that is a multiple (0, 15, 30, 45)
+  * `0-5` — Range, matches any value within the range (0, 1, 2, 3, 4, 5)
+  * `0-9/2` — Step values can be used in conjunction with ranges (0, 2, 4, 6, 8)
+
+Each part may have multiple rules, where rules are separated by a comma. The allowed values for
+each field are as follows:
+
+| Field      | Allowed Values                               |
```

Comment:
Turned some stuff into tables here, I think it looks great.

![Uploading CleanShot 2024-11-07 at 17.28.34@2x.png…]()